Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -768,7 +780,7 @@ extension TabDisplayManager: TabManagerDelegate {
 
         updateWith(animationType: .addTab) { [unowned self] in
             // place new tab at the end by default unless it has been opened from parent tab
-            var indexToPlaceTab = dataStore.count - 1 > 0 ? dataStore.count - 1 : 0
+            var indexToPlaceTab = dataStore.count - 1 > 0 ? dataStore.count : 0
```

Comment:
Curious about this - how did `-1` benefit us before?

Reply:
@nishant2718 I'm not sure 😅 @nbhasin2 do you know?

Reply:
Discussed with @nbhasin2, this is fine for now since it's using WeakList, so it doesn't insert, but append instead. At some point we'll remove WeakList and we'll need to be careful with that.